kymatio/utils.py

```diff
@@ -0,0 +1,53 @@
+from sklearn.base import BaseEstimator, TransformerMixin
```

Comment:
Maybe makes more sense to name this module `sklearn` instead of `utils`?

Reply:
Yes. Definintely. It should be a sklearn file, and we should do from kymatio.sklearn import ...

Reply:
Yes, and (as you said) it would be good to allow this to be instantiated using the `ScatteringEntry` class. However, this is not possible right now since we don't follow the same conventions (wrapping the `Scattering` object, not having a separate class for each dimension).

Reply:
Having 3 separate transformers might make sense. I do prefer that over havign the use create their own scattering objects, just thought it would look messy

Now about the np-only framework, Maybe I am a bit naive but I think for many cases cpu scatteirng is slow to the point of not being useful, so if we dont expose the gpu acceleration to a sklearn user we might never really get any sklearn users?